app/components/pages/Home.jsx

```diff
@@ -74,17 +77,14 @@ class Home extends React.Component {
         }
 
         <section>
-          <h2>Contacto</h2>
-
-          <p>
-            Se quiserdes contactar as pessoas responsaveis pela manutencao da
-            plataforma, com queixas, sugestoes, ou qualquer cena, mandai email
-            para <a href={`mailto:${email}`}>{email}</a>.
-          </p>
+          <h2>{t("contact.title")}</h2>
+          <p dangerouslySetInnerHTML={{ __html: t("contact.content", { email })}} />
```

Comment:
I'm fine with letting this go for now, but eventually I'd like to find a solution for this.

Reply:
I think this is actually the cleanest way to achieve this. We should only use for values we define ourselves, but the other solutions provided by the `react-i18next` (`<Trans>`, `<Interpolate />`) are completely obtuse: https://react.i18next.com/components/trans-component.html